client/app/components/saveable-form.hbs

```diff
@@ -12,12 +12,8 @@
       submittableChanges=submittableValidator.changeset
 
       Field=(component 'saveable-form/field'
-        saveableChanges=saveableValidator.changeset
-        submittableChanges=submittableValidator.changeset
-      )
-
-      Radio=(component 'radio-button'
```

Comment:
Oh okay, I see you deleted it. (Since the Radio here isn't needed at a top level anymore since it's accessed through Field)